domain/src/main/java/com/globant/equattrocchio/domain/GetSpecificImageUseCase.java

```diff
@@ -0,0 +1,22 @@
+package com.globant.equattrocchio.domain;
+
+
+import com.globant.equattrocchio.domain.enities.Image;
+import com.globant.equattrocchio.domain.service.ImagesServices;
+import io.reactivex.observers.DisposableObserver;
+
+public class GetSpecificImageUseCase extends UseCase<Image,Integer> {
+
+        private ImagesServices imagesServices;
+
+        public GetSpecificImageUseCase(ImagesServices imagesServices) {
+            super();
+            this.imagesServices = imagesServices;
+        }
+
+        @Override
+        void buildUseCaseObservable(DisposableObserver<Image> observer, Integer id) {
```

Comment:
same here

Reply:
this is an override method from UseCase, i cant put a int type there because the parametrization no accept primitive types